src/colvargrid.h

```diff
@@ -1474,6 +1660,15 @@ class colvar_grid_gradient : public colvar_grid<cvm::real>
     }
   }
 
+  /// \brief Report the offset between the lower end of the bin and the current value of variable i
+  inline cvm::real current_offset(int const i) const
```

Comment:
Maybe the function name could be a bit more descriptive, since elsewhere you have started indexing a colvargrid as a linear array with an integer?

Reply:
Also unused now. This should become an ingredient for a new function, see below.